remo/config.py

```diff
@@ -1,9 +1,18 @@
 import json
 import os
+from pathlib import Path
+
+REMO_HOME = os.getenv('REMO_HOME', str(Path.home().joinpath('.remo')))
 
 
 class Config:
-    __slots__ = ['port', 'server', 'user_name', 'user_email', 'user_password', 'conda_env', 'viewer']
+    """
+    Remo Config
+
+    This class is used to initialise various settings.
+    #TODO: add description of how those are initiliased from code vs from config file
```

Comment:
@vmanilo not clear here how we use these values. Do we try to read from a config, and if not present, use these values here?

Reply:
yes, we have some default values, if they missing in config

but I don't expect that user need to use this class